app/reducers/ProductsReducer.jsx

```diff
@@ -0,0 +1,34 @@
+import axios from 'axios'
+
+const initialState = {
+  products: [{name: 'blah', imageURL: 'whatever', price: 10000}]
+}
+
+// CONSTANTS
+export const LOAD_PRODUCTS = 'LOAD_PRODUCTS'
+
```

Comment:
The getProducts function should be under your Actions section; constants should only be the strings you're using as action types